flake.nix

```diff
@@ -51,8 +51,6 @@
               };
             in pkg: if lib.hasAttr pkg mapping then mapping.${pkg} else pkgs.${pkg})
             (lib.flatten (lib.attrValues config.programs.nixvim.plugins.lint.lintersByFt));
-
-            plugins.lsp.servers.angularls.package = pkgs.callPackage ./pkgs/angular-language-server { };
```

Comment:
Why did you drop the server altogether? Don't you need to specify the nixpkgs package here?

Reply:
Right, nixvim needs to update, too. Let me create a PR. https://github.com/nix-community/nixvim/pull/2506